fulp_modules/bloodsuckers/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -0,0 +1,479 @@
+//									IDEAS		--
+//					An object that disguises your coffin while you're in it!
+//
+//					An object that lets your lair itself protect you from sunlight, like a coffin would (no healing tho)
+
+
+
+// Hide a random object somewhere on the station:
+//		var/turf/targetturf = get_random_station_turf()
+//		var/turf/targetturf = get_safe_random_station_turf()
+
+
+
+
+// 		CRYPT OBJECTS
+//
+//
+// 	PODIUM		Stores your Relics
+//
+// 	ALTAR		Transmute items into sacred items.
+//
+//	PORTRAIT	Gaze into your past to: restore mood boost?
+//
+//	BOOKSHELF	Discover secrets about crew and locations. Learn languages. Learn marial arts.
+//
+//	BRAZER		Burn rare ingredients to gleen insights.
+//
+//	RUG			Ornate, and creaks when stepped upon by any humanoid other than yourself and your vassals.
+//
+//	X COFFIN		(Handled elsewhere)
+//
+//	X CANDELABRA	(Handled elsewhere)
+//
+//	THRONE		Your mental powers work at any range on anyone inside your crypt.
+//
+//	MIRROR		Find any person
+//
+//	BUST/STATUE	Create terror, but looks just like you (maybe just in Examine?)
+
+
+//		RELICS
+//
+//	RITUAL DAGGER
+//
+// 	SKULL
+//
+//	VAMPIRIC SCROLL
+//
+//	SAINTS BONES
+//
+//	GRIMOIRE
+
+
+// 		RARE INGREDIENTS
+// Ore
+// Books (Manuals)
+
+
+// 										NOTE:  Look up AI and Sentient Disease to see how the game handles the selector logo that only one player is allowed to see. We could add hud for vamps to that?
+//											   ALTERNATIVELY, use the Vamp Huds on relics to mark them, but only show to relevant vamps?
+
+
+/obj/structure/bloodsucker
+	var/mob/living/owner
+
+/*
+/obj/structure/bloodsucker/bloodthrone
+	name = "wicked throne"
+	desc = "Twisted metal shards jut from the arm rests. Very uncomfortable looking. It would take a sadistic sort to sit on this jagged piece of furniture."
```

Comment:
```suggestion
	desc = "Twisted metal shards jut from the arm rests. Very uncomfortable looking. It would take a masochistic sort to sit on this jagged piece of furniture."
```